_config.yml

```diff
@@ -113,6 +113,14 @@ defaults:
       path: 'content/**/*/*._zh.md'
     values:
       lang: zh
+  - scope:
+      path:            'admin/'
+    values:
+      sitemap:         false
+  - scope:
+      path:            'docs/unlinked/*'
+    values:
+      sitemap:         false      
```

Comment:
Is the extra spacing intentional?

If this is an alignment thing, I personally find that sort of formatting to be a hassle to maintain, create unwieldy diffs, and difficult to enforce consistently (also see inconsistency with the rest of the file).

```suggestion
  - scope:
      path: 'admin/'
    values:
      sitemap: false
  - scope:
      path: 'docs/unlinked/*'
    values:
      sitemap: false      
```